pkg/operator/ceph/object/user/controller.go

```diff
@@ -295,6 +295,8 @@ func (r *ReconcileObjectStoreUser) createOrUpdateCephUser(u *cephv1.CephObjectSt
 		}
 	}
 
+	logger.Debugf("user capabilities(id: %s, caps: %#v, user caps: %s, op mask: %s",
```

Comment:
Add this log to just after `//Update max cap if necessary` comment. IMHO all the logs in the PR need to Trace than Debug